tests/test_parse_osm.py

```
@@ -458,9 +504,10 @@ async def test_set_place_name_no_dupe_param(
         ),
     ],
 )
-async def test_finalize_last_place_name_variants(osm_parser, case, existing_attrs, should_set):
+async def test_finalize_last_place_name_variants(
+    osm_parser: OSMParserFactory, case: str, existing_attrs: Attrs, should_set: bool
```

Ruff (0.15.12)

[warning] Unused function argument: `case`

(ARG001)